### PR TITLE
2.2.0 Readme Dev

### PR DESCRIPTION
**Pull request by YamatoSecurity**
This PR is for the readme updates for the next version. I will keep it as a draft until we are ready to release.

### CODE REVIEW

**Comment by YamatoSecurity** [comment]
@fukusuket @hitenkoku 
I updated the readme documentation. It should be ready now for the 2.2.0 release. Please check it over when you can.

File: README-Japanese.md [hunk 1]
```diff
@@ -1177,18 +1315,24 @@ Windows機での悪性な活動を検知する為には、デフォルトのロ
...
 
 少なくとも、私たちのツールを気に入っていただけたなら、GitHubで星を付けて、あなたのサポートを表明してください。
 
 # バグの報告
 
-見つけたバグを[こちら](https://github.com/Yamato-Security/hayabusa/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5Bbug%5D)でご連絡ください。報告されたバグを喜んで修正します！
+見つけたバグを[こちら](https://github.com/Yamato-Security/hayabusa/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5Bbug%5D)でご連絡ください。
+報告されたバグを喜んで修正します！
+
+Hayabusaルールの問題点（誤検出、バグ等々）を発見された方は、hayabusa-rulesのGitHubの[Issues]((https://github.com/Yamato-Security/hayabusa-rules/issues/new)ページにご報告ください。
```
Comment (fukusuket) [inline README-Japanese.md:1328]:
```suggestion
Hayabusaルールの問題点（誤検出、バグ等々）を発見された方は、hayabusa-rulesのGitHubの[Issues](https://github.com/Yamato-Security/hayabusa-rules/issues/new)ページにご報告ください。
```

File: README-Japanese.md [hunk 2]
```diff
@@ -597,198 +805,158 @@ Output:
...
-* 全てのフィールド情報も含めて１つのCSVファイルにエクスポートして、Excel、Timeline Explorer、Elastic Stack等でさらに分析することができる(注意: `super-verbose`プロファイルを使すると、出力するファイルのサイズがとても大きくなる！):
+普段は次のように実行します: `hayabusa.exe update-rules`
 
-```
-hayabusa-2.1.0-win-x64.exe csv-timeline -d .\hayabusa-sample-evtx -o results.csv -P super-verbose
-```
+## `level-tuning`コマンド
 
-* タイムラインをJSON形式で保存する:
+The `level-tuning` command will let you tune the alert levels for rules, either raising or decreasing the risk level according to your environment.
```
Comment (fukusuket) [inline README-Japanese.md:865]:
```suggestion
`level-tuning`コマンドを使用すると、環境に応じてリスク レベルを上げたり下げたりして、ルールのアラート レベルを調整できます。
```

**Review by fukusuket — CHANGES_REQUESTED** [review]
I made a review comment :) please check it 🙏

**Comment by YamatoSecurity** [comment]
@fukusuket Thank you so much for the quick review! I comitted your suggestions.

**Review by fukusuket — APPROVED** [review]
Thank you! LGTM :)

**Review by hitenkoku — APPROVED** [review]
LGTM

**Comment by YamatoSecurity** [comment]
Thank you both for checking!

**Pull request merged**